sphinxcontrib/jupyter/__init__.py

```diff
@@ -17,13 +17,15 @@ def setup(app):
     app.add_config_value("jupyter_lang_synonyms", [], "jupyter")
     app.add_config_value("jupyter_drop_solutions", True, "jupyter")
     app.add_config_value("jupyter_drop_tests", True, "jupyter")
+    app.add_config_value("jupyter_slide", True, "jupyter")  
```

Comment:
@FelipeMaldonado is this required anymore?

Reply:
Let me test what happen if I remove it.

Reply:
It wasn't required, so I removed it (in `translate_all`, `translate_code`, and `__init__`)